avm-transpiler/src/transpile.rs

```diff
@@ -834,6 +834,28 @@ fn handle_black_box_function(avm_instrs: &mut Vec<AvmInstruction>, operation: &B
                 ],
             });
         }
+        BlackBoxOp::EmbeddedCurveAdd {
+            input1_x,
+            input1_y,
+            input1_infinite,
+            input2_x,
+            input2_y,
+            input2_infinite,
+            result,
```

Comment:
optionally, if you have to change sth else, you might want to call these `input1_x_offset`, etc? I think we do that with others? (maybe in rust you can do: `input1_x: input1_x_offset` or similar?